Sources/ArcGISToolkit/Components/OfflineMapAreasView/PreplannedMapModel.swift

```diff
@@ -17,15 +17,50 @@ import ArcGIS
 
 /// An object that encapsulates state about a preplanned map.
 @MainActor
-class PreplannedMapModel: ObservableObject, Identifiable {
+public class PreplannedMapModel: ObservableObject, Identifiable {
```

Comment:
I might have missed it - why it needs to be public?

Ideally, we don't expose any internal logic within the data model, and only expose the APIs via the top level view - `OfflineMapAreasView`.

If there is a specific case we need to make the data model public, we can discuss to see if there is a better way to expose the API.

```suggestion
class PreplannedMapModel: ObservableObject, Identifiable {
```

Reply:
This class can be internal.